src/parser/jobs/nin/modules/Ninki.js

```diff
@@ -104,4 +121,26 @@ export default class Ninki extends Module {
 			</Trans>,
 		}))
 	}
+
+	output() {
+		const _ninkiColor = Color(JOBS.NINJA.colour)
+
+		/* eslint-disable no-magic-numbers */
+		const chartdata = {
+			datasets: [
+				{
+					label: 'Ninki',
```

Comment:
I feel like this should be i18n'd as well, but I'm not actually sure how, so I'm fine brushing that one under the carpet unless we have another chart somewhere that does it to use as an example. RDM didn't, I can say that much (I based my version on it).

Reply:
I can't find one that does this either, so I'm going to skip it for now.